bin/pubber

```diff
@@ -31,29 +31,24 @@ else
         if [[ $option == *"="* ]]; then
             k=$(echo $option | cut -d'=' -f1)
             v=$(echo $option | cut -d'=' -f2)
-            options[$k]=$v
         else
-            options[$option]=$option
+            k=$option
+            v=$option
```

Comment:
I'm wondering if this should be a boolean.  Ideally we can be more semantic about this, and having it as a boolean would allow us to make sure people are using options correctly...

Reply:
Agree, turned it into a boolean with correct type assignment for checking